src/RSCPL.xml

```diff
@@ -457,6 +460,7 @@
          responsibility on an equitable basis.</p>
        </li>
       </list>
+      
```

Comment:
You're introducing trailing whitespace here (and in a few other places) with this commit:

```
$ git show --oneline --check dfd45e2c
dfd45e2 Update RSCPL.xml
src/RSCPL.xml:60: trailing whitespace.
+    
src/RSCPL.xml:111: trailing whitespace.
+        
src/RSCPL.xml:463: trailing whitespace.
+      
```